src/components/StoryList/index.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+import * as MainSiteStyles from '../../globals/mainsiteGlobalStyles'
+import Header from './Header'
+import Story from './Story'
+
+/**
+ * Story List Properties
+ */
+
+interface StoryContent {
+  title: string
+  text: string
+  link: string
+}
+
+interface ImageContent {
+  src: string
+  alt: string
+}
+
+enum StoryListType {
+  Popular = 'popular',
+  Print = 'print',
+}
+
+interface StoryListProps {
+  /** A list of the most popular stories. The list should contain exactly 3 stories. */
```

Comment:
Why the limit on 3 stories?

Reply:
On figma, those components only had 3 stories on them, so I assumed it was only 3.

Reply:
let's make it arbitrary because why not? we'll likely be using three stories, but I don't see why we should just be limited to that.